### DISCUSSION TITLE
exception info

### DISCUSSION THREAD

**Discussion by stefano-st** [cenobites/flask-jsonrpc#474] [Q&A] [answered]
Hi mr Nicholas,
i need some info about this library.
For security purpose , when I send a malformed request to the server , the exception raised show to many information about the remote server structure. There is any way to limit the details?
thanks
Regards
Stefano

**Comment by nycholas** [comment, marked as answer]
@stefano-st This question passed overhead, sorry!

The exception raised will only show the stack trace of the error (the information mentioned) if the server is in debug mode, in another case, the information on the response will be the following attributes: code, message, and data. For example:

```
{
    'error': {
        'code': -32600,
        'data': {
            'message': "The method 'app.fn4' doesn't allow Notification Request object (without an 'id' member)"
        },
        'message': 'Invalid Request',
        'name': 'InvalidRequestError',
    },
    'id': None,
    'jsonrpc': '2.0',
}
```

Thank you.
